src/pages/userPage/ActiveMembership/PlastDegree/ModalAddPlastDegree.tsx

```diff
@@ -63,7 +64,12 @@ const ModalAddPlastDegree = (props: ModalAddPlastDegreeProps) => {
     <Modal
       visible={props.visibleModal}
       onCancel={handleCancel}
-      title={isUserDataLoaded ? `Прийняти до уладу ${fullName} (${age} р.)` : "Завантаження..."}
+      title={
+        isUserDataLoaded
+          ? props.isEditing
```

Comment:
isEditing field holds very little meaning to other programmers. Can you rename it to something more meaningful? "isChangingDegree" or "userHasDegree" for example